Art/gauravsaxena816/include.js

```diff
@@ -0,0 +1,43 @@
+let cards = [
+  {
+    artName: 'Loader Animtion',
+    pageLink: './Art/gauravsaxena816/loader.html',
+    imageLink: './Art/gauravsaxena816/loader.gif',
+    author: 'Gaurav Saxena',
+    githubLink: 'https://github.com/Gauravsaxena816'
+  }
+];
+
+// +--------------------------------------------------------------------------------+
+// +                                                                                +
+// +                  YOU DO NOT NEED TO CHANGE ANYTHING BELOW THIS                 +
+// +                                                                                +
+// +--------------------------------------------------------------------------------+
+
+// Creates cards from the array above
+// You don't need to modify this
+let contents = [];
+Shuffle(cards).forEach((c) => {
+  contents.push([
+    `<li class="card">` +
+      `<a href='${c.pageLink}'>` +
+      `<img class="art-image" src='${c.imageLink}' alt='${c.artName}' />` +
+      `</a>` +
+      `<div class="flex-content">` +
+      `<a href='${c.pageLink}'><h3 class="art-title">${c.artName}</h3></a>` +
+      `<p class='author'><a href="${c.githubLink}" target="_blank"><i class="fab fa-github"></i> ${c.author}</a> </p>` +
+      `</div>` +
+      `</li>`
+  ]);
+});
+
+document.getElementById('cards').innerHTML = contents;
+
+function Shuffle(o) {
+  for (
+    var j, x, i = o.length;
+    i;
+    j = parseInt(Math.random() * i), x = o[--i], o[i] = o[j], o[j] = x
+  );
+  return o;
+}
```

Comment:
Changes need to be made to the root `include.js` instead of adding an additional file into your folder.